src/odemis/gui/main_xrc.py

```diff
@@ -3935,7 +3935,7 @@ def __init_resources():
             </object>
             <object class="sizeritem">
               <object class="ToolBar" name="sparc_acq_toolbar">
```

Comment:
I'm not sure it makes sense to have it in the SPARC acquisition, as these labels will not be stored in the acquisition. And if it's on the SPARC, we probably should also have it on the SECOM. Let's discuss it with Toon or Sangeetha.